### DISCUSSION TITLE
Best way to start helping

### DISCUSSION THREAD

**Discussion by and-rad** [bevyengine/bevy#14529] [Q&A]
Let's assume someone has never written a line of Rust in their life, but has a decade of experience with different engines, both closed and open source, and has contributed to / hacked away at those engines in the past. What would be a good/productive way to start contributing to the Bevy project in those circumstances?

I'm thinking of contributing documentation, confirming issues, reporting bugs. Is there another area, maybe specific to the current state of the engine, that is in need of some assistance?

**Comment by bugsweeper** [comment]
I think in those circumstances the best what you can do is bring your expierence in feature proposals. It's hard to do if you know nothing about Bevy, so you could try to open some tracking issues like [this](https://github.com/bevyengine/bevy/issues/10572) or discussions about feature proposals like [this](https://github.com/bevyengine/bevy/discussions/14437) and give proposals about features, which could make easier migrating from other engine to Bevy. Reading them could also improve your knowledge about Bevy.

**Comment by JMS55** [comment]
Most development is coordinated on the bevy discord. I suggest joining the discord and asking in the #engine-dev channel for where to start, and the maintainers will be able to help point you in the right direction based on your interests and experience.
